packages/edit-post/src/components/options-modal/options/enable-custom-fields.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { withSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import BaseOption from './base';
+
+class EnableCustomFieldsOption extends Component {
+	constructor( { isEnabled } ) {
+		super( ...arguments );
+
+		this.toggleCustomFields = this.toggleCustomFields.bind( this );
+
+		this.state = { isEnabled };
```

Comment:
Nitpick: this state is called isChecked in DeferredOption.

Reply:
2c11c00ac6d9257b0df9ef63e7f8425906be9152